xivo/auth_verifier.py

```diff
@@ -70,6 +70,36 @@ def __init__(self, token, required_access=None):
         )
 
 
+class Invalid_Token_Exception(rest_api_helpers.APIException):
```

Comment:
You should not use this style for the name
```suggestion
class InvalidTokenAPIException(rest_api_helpers.APIException):
```